### DISCUSSION TITLE
Hostname or IP in page title

### DISCUSSION THREAD

**Discussion by kenara** [webmin/webmin#2087] [Ideas]
I have lots of cloud servers all using Webmin and often have many of them open in browser tabs. It would be nice to be able to mouse over the tab without opening it and identify the server. This would be possible if the hostname (truncated if necessary) or IP address could be included in the page title. That way, instead of 

`Dashboard - Webmin 2.105 (Ubuntu Linux 22.04.3)` it might say

 `Dashboard - Mapserver - Webmin 2.105 (Ubuntu Linux 22.04.3)` or some such..

**Comment by jcameron** [comment]
This can be configured at Webmin -> Webmin Configuration -> User Interface

**Reply by kenara** [reply]
I don't find it under Webmin -> Webmin Configuration -> User Interface
There, I only see: 

- Hostname to display in Webmin; 
- Send feedback to, and
- Allow sending of feedback?

It's not under Module titles although that seemed like a candidate. That one prepends the custom name to the title of the given module but "Dashboard" remains unchanged.

I'll keep looking...

**Reply by iliaross** [reply]
This option in Authentic theme is located under **Theme Configuration: General defaults** page, which you can find it under **Document title format** name.

**Reply by kenara** [reply]
Wow! Great, thanks!
